src/content/app/genome-browser/components/drawer/Drawer.tsx

```diff
@@ -22,11 +22,18 @@ import DrawerBookmarks from './drawer-views/DrawerBookmarks';
 import GeneSummary from './drawer-views/gene-summary/GeneSummary';
 import TranscriptSummary from './drawer-views/transcript-summary/TranscriptSummary';
 import VariantGroupLegend from './drawer-views/variant-group-legend/VariantGroupLegend';
+import { CircleLoader } from 'src/shared/components/loader';
 
 import { getActiveDrawerView } from 'src/content/app/genome-browser/state/drawer/drawerSelectors';
 
 import styles from './Drawer.scss';
 
+export const Spinner = () => (
+  <div className={styles.spinnerContainer}>
+    <CircleLoader />
+  </div>
+);
```

Comment:
I am a bit worried that `Drawer` imports components (`GeneSummary`, `TranscriptSummary`), which now depend on something that `Drawer` exports.

Is this a circular dependency? Maybe not quite; but it feels safer to put the spinner in its own `DrawerSpinner` file, next to `Drawer.tsx`